frontend/public/style/_vars.scss

```diff
@@ -27,6 +27,13 @@ $pf-v6-global-gutter-x: calc(
 $pf-v6-global-gutter-y: var(--pf-t--global--spacer--md); // Matches pf-v6-c-page__main-section
 
 // == Custom variables
+
+// The height+padding of a PatternFly standard button.
+$co-button-height: calc(
+  var(--pf-t--global--font--size--body--default) * var(--pf-t--global--font--line-height--body) +
+    var(--pf-t--global--spacer--control--vertical--default) * 2
+);
+
```

Comment:
Nit, remove the extra line.

Reply:
addressed in https://github.com/openshift/console/pull/14995